extensions/auth/portability-auth-oauth2/src/main/java/org/datatransferproject/auth/oauth2/OAuth2Config.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.datatransferproject.auth.oauth2;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public interface OAuth2Config {
+
+  String getServiceName();
+
+  String getAuthUrl();
+
+  String getTokenUrl();
+
+  Map<String, List<String>> getExportScopes();
```

Comment:
Does ordering or duplicates matter? Should it be a Set or OrderedSet instead?

Reply:
Add comment including the first value is 'transferType' as defined in the auth data generators and elsewhere.

Reply:
Order does not seem to matter, but I can't imagine duplicates would be a good thing.